source/index.tmpl

```diff
@@ -66,217 +66,9 @@
             <button>&rarr; Signup for Updates &larr;</button>
           </a>
         </header>
-        </br>
+        <br> 
       </div>
     </div>
-    <!-- END PAGE | Welcome -->

```

Comment:
leave this commented out and not deleted, it will be uncommented in about a month when we have speakers